From 6dc4c8efb164f1f5ed79f5daaa952059d851db34 Mon Sep 17 00:00:00 2001
From: Nicola Fontana <ntd@entidi.it>
Date: Thu, 11 Feb 2016 12:58:34 +0100
Subject: [PATCH 1/9] Use PKG_CONFIG when running g-ir-scanner

While cross-compiling, "$PKG_CONFIG" is not "pkg-config", so do not
assume it when calling g-ir-scanner to build up the base gir files.

https://bugzilla.gnome.org/show_bug.cgi?id=761983
---
 Makefile.introspection | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.introspection b/Makefile.introspection
index cd7bd1b..1812df9 100644
--- a/Makefile.introspection
+++ b/Makefile.introspection
@@ -84,7 +84,7 @@ _gir_silent_compiler = $(_gir_silent_compiler_$(V))
 _gir_silent_compiler_ = $(_gir_silent_compiler_$(_gir_verbosity))
 _gir_silent_compiler_0 = @echo "  GICOMP   $(1)";
 
-_gir_default_scanner_env = CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" CC="$(CC)"
+_gir_default_scanner_env = CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" CC="$(CC)" PKG_CONFIG="$(PKG_CONFIG)"
 
 #
 # Creates a GIR by scanning C headers/sources
-- 
2.7.1

